tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -0,0 +1,66 @@
+*** Settings ***
+Documentation    Collection of keywords to interact with Model Serving
+Resource       ../../../../Page/Components/Components.resource
+Resource       ../../../../Common.robot
+
+*** Variables ***
+${MS_HEADER_XP}=     xpath=//h1[text()="Deployed models"]
+
+*** Keywords ***
+Open Model Serving Home Page
+    [Documentation]    Verifies submenu Settings > "Model Serving" is visible and navigates to page
+    Page Should Contain    Model Serving
+    Click Link      Model Serving
+    Wait Until Page Contains    Manage and view the health and performance of your deployed models.    timeout=30
+    Wait Until Page Contains Element    ${MS_HEADER_XP}    timeout=30
+    Maybe Wait For Dashboard Loading Spinner Page
+
+Serve Model
+    [Documentation]
+    [Arguments]    ${project_name}    ${model_name}    ${framework}    ${existing_data_connection}=${TRUE}
+    ...    ${data_connection_name}    ${model_path}
+    Click Button    Serve model
+    Wait Until Page Contains    xpath://h1[.="Deploy model"]
+    Select Project    ${project_name}
+    Set Model Name    ${model_name}
+    Select Framework    ${framework}
+    IF    ${existing_data_connection}==${TRUE} 
+        Select Radio Button    group_name=model-location    value=existing-data-connection-radio
+        Select Existing Data Connection    ${data_connection_name}
+        Set Folder Path    ${model_path}
+    ELSE
+        Select Radio Button    group_name=model-location    value=new-data-connection-radio
+        # Set Up New Data Connection
+    END
+    Click Button    Configure
+
+Select Project
+    [Documentation]
+    [Arguments]    ${project_name}
+    Click Element    xpath://span[.="Project"]/../../..//button[@aria-label="Options menu"]
+    Click Element    xpath://li/button[.="${project_name}"]
```

Comment:
you could create a parameterized keyword to do selection, since I see the same 2 lines are used for other selection tasks (e.g., Select Framework or Select Existing Data Connection)

Reply:
Yeah, I was waiting to see if an id or other would be added to these elements